client/src/useJobHelper.test.js

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, test } from "@jest/globals";
+
+import { getAgeString } from "./useJobHelper";
+
+describe("useJobHelper", () => {
+    test.each([
+        [1, "1 minute old"],
+        [2, "2 minutes old"],
+        [61, "1 hour old"],
+        [121, "2 hours old"],
+        [1440, "1 day old"],
```

Comment:
Should we have a test for 2 days old? Seems like the only thing missing from the pattern here.